cardano-cli/test/cardano-cli-golden/Test/Golden/Byron/Tx.hs

```diff
@@ -34,14 +32,15 @@ hprop_byronTx_legacy = propertyOnce $ H.moduleWorkspace "tmp" $ \tempDir -> do
     , "--txout", "(\"2657WMsDfac6eFirdvKVPVMxNVYuACd1RGM2arH3g1y1yaQCr1yYpb2jr2b2aSiDZ\",999)"
     ]
 
-  compareByronTxs createdTx expectedTx
+  void $ compareByronTxs createdTx expectedTx
 
 hprop_byronTx :: Property
-hprop_byronTx = propertyOnce $ H.moduleWorkspace "tmp" $ \tempDir -> do
-  signingKey <- noteInputFile "test/cardano-cli-golden/files/input/byron/keys/byron.skey"
-  expectedTx <- noteInputFile "test/cardano-cli-golden/files/input/byron/tx/normal.tx"
-  createdTx <- noteTempFile tempDir "tx"
-  void $ execCardanoCLI
+hprop_byronTx = propertyOnce $ localWorkspace $ do
+  signingKey <- jotPkgInputFile "test/cardano-cli-golden/files/input/byron/keys/byron.skey"
+  expectedTx <- jotPkgInputFile "test/cardano-cli-golden/files/input/byron/tx/normal.tx"
+  createdTx <- jotTempFile "tx"
```

Comment:
We can ask for a temp file without supplying the `tempDir` because it can get it from `Reader ProjectRoot`.